internal/indexnode/task.go

```diff
@@ -217,6 +217,12 @@ func (it *IndexBuildTask) saveIndexMeta(ctx context.Context) error {
 			panic(errMsg)
 		}
 
+		if it.internalErr != nil {
```

Comment:
This line will never be reached because of panic of line 217.

Reply:
yes, but if a panic occurs on line 217, the index meta of this task  does not need to be stored in etcd anymore (it will fail).

Reply:
and indexcoord will reassign this task.